tests/rust/enum.rs

```diff
@@ -1,98 +1,106 @@
 enum Opaque {
     Foo(i32),
-    Bar
+    Bar,
 }
 
-#[repr(u32)]
+#[repr(u64)]
```

Comment:
Would've been easier to just add a new enum instead, but nbd.

Reply:
I figured you might want me to keep the same ordering in the test (then again, it doesn't really matter).